HSPI/App.config

```diff
@@ -1,7 +1,6 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-
+<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup>
-        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5.2" />
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.1"/>
```

Comment:
The supported runtime is 4.7.1 here but all your Nuget packages are targeting 4.6.1. Is this right?